client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -260,6 +261,11 @@ function CellIcons({ support }: { support: bcd.SupportStatement | undefined }) {
   return (
     <div className="bc-icons">
       {supportItem.prefix && <Icon name="prefix" />}
+      {supportItem.hasOwnProperty("notes") && <Icon name="footnote" />}
+      {!supportItem.hasOwnProperty("notes") &&
+        asList(support).some((s) => s && s.hasOwnProperty("notes")) && (
+          <Icon name="chevron" />
+        )}
```

Comment:
The check that corresponds to this below is actually `Array.isArray(browserSupport)`

Reply:
As for why this is important, I'm not totally sure it is. I was just trying to match:

![This](https://user-images.githubusercontent.com/159415/156348453-a65414cc-9244-461e-8915-cdb6b3db3fb3.png)

Reply:
> If we feels that's not needed, I can remove this check.

I believe we don't need to restore the chevron at all:

- In the old design, the chevron signaled that you can expand a cell. You could only expand cells with historical notes, so that was important.
- Since the redesign, you can now expand _every_ cell, so adding a chevron to some cells might mislead users into thinking they can _only_ expand these.

For now, let's stick with this:

- If the current support item has a note, add the notes icon (like you already implemented).
- If an older (i.e. outdated) support item has a note, do _not_ add any icon or chevron.

Does that sound okay for you?

Reply:
I think that's okay, so I'll implement it. If the browser version visible in the un-expanded table was the older version, I could see more of an argument for the chevron. Since @dipikabh commented about the chevron though, I'd be curious to hear if they think there's a need for it.

I could see the argument for adding it though: While you can expand every cell, no one thinks they need to expand a cell with just a checkmark. The chevron would tell you that more info exists. Maybe a user needs to support a browser that's older then the version listed in the un-expanded table. They might think they can't use that feature because they assume it wasn't added until the listed version, when really there may just be some minor bug that they could easily work around, but they wouldn't know that at a glance.

Reply:
I think I agree with @danielhjacobs here about the need to be able to expand to look for more info and indicating which cells offer more info. My comment was more to do with the chevron icon itself and not for removing that functionality.